gatsby-config.js

```diff
@@ -620,8 +620,18 @@ module.exports = {
                                         path: "/home/base/mobile-core/identity/api-reference"
                                     },
                                     {
-                                        title: "Push identifier sync",
-                                        path: "/home/base/mobile-core/identity/push-sync"
+                                        title: "Tutorials",
+                                        path: "/home/base/mobile-core/identity/tutorials",
+                                        pages: [
+                                            {
+                                                title: "Push identifier sync",
+                                                path: "/home/base/mobile-core/identity/tutorials/push-sync"
+                                            },
+                                            {
+                                                title: "Visitor tracking",
+                                                path: "/home/base/mobile-core/identity/tutorials/visitor-tracking"
```

Comment:
renaming suggestion (similar to some of the experience league tutorials) + reorder to have this one first

Mobile to web ID sharing 
link: tutorials/id-sharing